Lib/typing.py

```diff
@@ -2093,6 +2094,23 @@ def cast(typ, val):
     return val
 
 
+def assert_type(val, typ, /):
+    """Assert (to the type checker) that the value is of the given type.
+
+    When the type checker encounters a call to assert_type(), it
+    emits an error if the value is not of the specified type::
+
+        def greet(name: str) -> None:
+            assert_type(name, str)  # ok
+            assert_type(name, int)  # type checker error
+
+    At runtime this returns the first argument unchanged and otherwise
+    does nothing.
+
+    """
```

Comment:
Nit: Unusual to have a blank line preceding a closing """. Suggest removing it.

Reply:
```suggestion
    """
```

Reply:
Thanks!